mlir/lib/Dialect/LLVMIR/IR/LLVMTypeSyntax.cpp

```diff
@@ -51,41 +51,6 @@ static StringRef getTypeKeyword(Type type) {
       });
 }
 
-/// Prints a structure type. Keeps track of known struct names to handle self-
-/// or mutually-referring structs without falling into infinite recursion.
-static void printStructType(AsmPrinter &printer, LLVMStructType type) {
-  FailureOr<AsmPrinter::CyclicPrintReset> cyclicPrint;
-
-  printer << "<";
-  if (type.isIdentified()) {
-    cyclicPrint = printer.tryStartCyclicPrint(type);
-
-    printer << '"' << type.getName() << '"';
-    // If we are printing a reference to one of the enclosing structs, just
-    // print the name and stop to avoid infinitely long output.
-    if (failed(cyclicPrint)) {
-      printer << '>';
-      return;
-    }
-    printer << ", ";
-  }
-
-  if (type.isIdentified() && type.isOpaque()) {
-    printer << "opaque>";
-    return;
-  }
-
-  if (type.isPacked())
-    printer << "packed ";
-
-  // Put the current type on stack to avoid infinite recursion.
-  printer << '(';
-  llvm::interleaveComma(type.getBody(), printer.getStream(),
-                        [&](Type subtype) { dispatchPrint(printer, subtype); });
-  printer << ')';
-  printer << '>';
-}
-
 /// Prints the given LLVM dialect type recursively. This leverages closedness of
```

Comment:
The comment can probably be shorten to:

/// Prints the given LLVM dialect type.

Or similar?